### PR TITLE
EPMRPP-35206 React: Not own dashboard view should be differ from the …

### PR DESCRIPTION
**Pull request by vitalii-chornyi**
…own dashboard view

### CODE REVIEW

**Comment by vitalii-chornyi** [comment]
> May be better to create const variable `isOwner` here to avoid several function calls?

Fixed

**Comment by vitalii-chornyi** [comment]
Fixed

**Comment by AmsterGet** [comment]
What about this caption (see the screenshot below)?
image

**Comment by vitalii-chornyi** [comment]
added the caption

File: app/src/pages/inside/dashboardItemPage/dashboardItemPage.jsx [hunk 1]
```diff
@@ -230,19 +239,35 @@ export class DashboardItemPage extends Component {
           <div className={cx('dashboard-item')}>
             <div className={cx('buttons-container')}>
               <div className={cx('nav-left')}>
-                <GhostButton icon={AddWidgetIcon} onClick={this.showWidgetWizard}>
-                  {formatMessage(messages.addNewWidget)}
-                </GhostButton>
-                <GhostButton icon={AddSharedWidgetIcon} onClick={this.showAddSharedWidgetModal}>
-                  {formatMessage(messages.addSharedWidget)}
-                </GhostButton>
+                {!isOwner && (
```
Comment (AmsterGet) [inline app/src/pages/inside/dashboardItemPage/dashboardItemPage.jsx]: Can we write it as: `isOwner ? (block with caption) : (<Fragment>buttons</Fragment>)`

File: app/src/pages/inside/dashboardItemPage/dashboardItemPage.jsx [hunk 2]
```diff
@@ -230,19 +239,35 @@ export class DashboardItemPage extends Component {
...
+                  <div className={cx('shared-caption')}>
+                    <span className={cx('globe-icon')}>{Parser(GlobeIcon)}</span>
+                    Dashboard is shared by {dashboard.owner}
```
Comment (AmsterGet) [inline app/src/pages/inside/dashboardItemPage/dashboardItemPage.jsx]: I think it should be a `react-intl` message to support localization

**Comment by vitalii-chornyi** [comment]
Fixed

**Pull request merged**